frontend/src/components/core/StreamlitDialog/SettingsDialog.tsx

```diff
@@ -88,6 +88,16 @@ export class SettingsDialog extends PureComponent<Props, UserSettings> {
         animate={this.props.animateModal}
         isOpen
         onClose={this.handleCancelButtonClick}
+        overrides={{
```

Comment:
Do we actually need to that the border-radius here as well? It is using the shared Modal implementation that also has the radius set, so we might be able to just remove this here.

Reply:
You're right, removed 👍